src/js/components/awardv2/idv/activity/chart/ActivityChart.jsx

```diff
@@ -106,18 +105,41 @@ export default class ActivityChart extends React.Component {
             const start = xScale(bar._startDate.valueOf()) + padding.left;
             const end = xScale(bar._endDate.valueOf()) + padding.left;
             const width = end - start;
+            // create a scale for obligated amount width using awarded amount
+            // and the awarded amount width
+            const obligatedAmountScale = scaleLinear()
+                .domain([0, bar._awardedAmount])
+                .range([0, width])
+                .nice();
+            // scale the abligated amount to create the correct width
+            const obligatedAmountWidth = obligatedAmountScale(bar._obligatedAmount);
             const yPosition = (height - 30) - yScale(bar._awardedAmount) - barHeight;
-            const description = `A ${bar.grandchild ? 'grandchild' : 'child'} award with a start date of ${bar.startDate}, an end date of ${bar.endDate}, an awarded amount of ${bar.awardedAmount}, and an obligated amount of ${bar.obligatedAmount}.`;
+            const awardBarDescription = `A ${bar.grandchild ? 'grandchild' : 'child'} award with a start date of ${bar.startDate}, an end date of ${bar.endDate}, adn an awarded amount of ${bar.awardedAmount}.`;
+            const obligatedBarDescription = `A ${bar.grandchild ? 'grandchild' : 'child'} award with a start date of ${bar.startDate}, an end date of ${bar.endDate}, and an obligated amount of ${bar.obligatedAmount}.`;
+            const awardedClassName = { 'activity-chart-bar__awarded-content': true };
+            const obligatedClassName = { 'activity-chart-bar__obligated-content': true };
             return (
-                <ActivityChartBar
-                    key={`bar-${bar._awardedAmount}-${index}`}
-                    index={index}
-                    height={barHeight}
-                    start={start}
-                    width={width}
-                    yPosition={yPosition}
-                    data={bar}
-                    description={description} />
+                <g key={`bar-${bar._awardedAmount}-${index}`}>
```

Comment:
I would add a single `description` (that includes both the obligated and awarded amount) to this wrapper element to avoid redundancy.